Examples/VisionExamples.xcodeproj/project.pbxproj

```diff
@@ -109,6 +112,14 @@
 			path = Resources;
 			sourceTree = "<group>";
 		};
+		3A21B816243348C700AA786C /* Basics */ = {
```

Comment:
Maybe we can call this "Code Snippets" and put on the same level as examples?

Reply:
Agree.